lib/base-controller.js

```diff
@@ -3,10 +3,12 @@
 var Controller = require('hmpo-form-wizard').Controller;
 var util = require('util');
 var _ = require('lodash');
+var mixins = require('./middleware/mixins');
 
 var BaseController = function BaseController(options) {
   this.confirmStep = options.confirmStep || '/confirm';
   Controller.apply(this, arguments);
+  this.use(mixins);
```

Comment:
This is how we use the template mixins - https://github.com/UKHomeOffice/gro/blob/master/apps/gro/index.js#L18 it is a middleware. Thought this would be the most consistent way - let me know if you think of something better though!